src/sentry/static/sentry/app/views/releases/releaseOverview.jsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import {Link} from 'react-router';
+
+import LoadingIndicator from '../../components/loadingIndicator';
+import LoadingError from '../../components/loadingError';
+import IssueList from '../../components/issueList';
+import FileChange from '../../components/fileChange';
+import CommitAuthorStats from '../../components/commitAuthorStats';
+import ReleaseProjectStatSparkline from '../../components/releaseProjectStatSparkline';
+import ApiMixin from '../../mixins/apiMixin';
+
+import {t} from '../../locale';
+
+
+const ReleaseOverview = React.createClass({
+  mixins: [ApiMixin],
+
+  getInitialState() {
+    return {
+      loading: true,
+      error: false,
+      projects: [],
+    };
+  },
+
+  componentDidMount() {
+    let {orgId, projectId, version} = this.props.params;
+
+    let path = `/projects/${orgId}/${projectId}/releases/${version}/commitfiles/`;
+    this.api.request(path, {
+      method: 'GET',
+      data: this.props.location.query,
+      success: (data, _, jqXHR) => {
+        this.setState({
+          error: false,
+          loading: false,
+          fileList: data,
+          pageLinks: jqXHR.getResponseHeader('Link')
+        });
+      },
+      error: () => {
+        this.setState({
+          error: true,
+          loading: false
+        });
+      }
+    });
+    this.getReleaseProjects();
+  },
+
+  getReleaseProjects() {
+    let {orgId, version} = this.props.params;
+    let path = `/organizations/${orgId}/releases/${version}/`;
+    this.api.request(path, {
+      method: 'GET',
+      success: (data, _, jqXHR) => {
+        this.setState({
+          projects: data.projects,
+          error: false,
+        });
+      },
+      error: () => {
+        this.setState({
+          error: true,
```

Comment:
Hmm, we do so much of this `loading/error` pattern ... wonder if we can abstract it into a utility component of some kind (just musing).